komodo/link.py

```diff
@@ -0,0 +1,56 @@
+import time
+import os
+import shutil
+
+import komodo
+
+age_limit = 7200
+
+
+def update_links(prefix):
+    # The first step is to scan the prefix directory and inspect all the
+    # directories in the folder. The directory names are split, and a
+    # dictionary:
+    #
+    #     release_list = {"2018.03" : [(1234, "2018.03-1234"),
+    #                                  (5678, "2018-03-5678")],
+    #                     "2018.10" : [(561, "2018.10.561"),
+    #                                  (777, "2018.10-777")]}
+    #
+    # This is dictionary where the release names like '2018.03' and '2018.10'
+    # are used as keys, the values in the dictionary are lists of tuples
+    # (timestamp, path_name).
+    ts_now = time.mktime( time.localtime() )
+    release_list = {}
+    for entry in os.listdir(prefix):
+        if os.path.islink(entry):
+            continue
+
+        if os.path.isdir(entry):
```

Comment:
please flip `if` and add `continue` to be consistent with the line above.  Also, I think `for→if→if→if` is too deep.

Reply:
Did not understand 100% what you mean here - but I have rewritten the second `if` to have an early `continue`